docs/using/troubleshooting.md

```diff
@@ -0,0 +1,307 @@
+# Troubleshooting Common Problems
+
+When troubleshooting, you may see unexpected behaviors or receive an error message. This section provides advice on
+how to identify and fix some of the most commonly encountered issues.
```

Comment:
```suggestion
When troubleshooting, you may see unexpected behaviors or receive an error message.
This section provides advice on how to identify and fix some of the most commonly encountered issues.
```

Reply:
Here is an example of how I like to put line breaks.

Reply:
So, some lines are longer, some are shorter, but in general they are still not too long, but most importantly - I don't cut the sentence in some random part, which in my opinion makes it easier to read.